### PR TITLE
Increase rule message size for postgres and mssql

### PR DESCRIPTION
**Pull request by VSEphpbb**
Means to fix: https://www.phpbb.com/customise/db/extension/boardrules/support/topic/144706

### CODE REVIEW

**Comment by rxu** [comment]
Reviewed, tested locally, working fine for me.

**Comment by VSEphpbb** [comment]
@michaelcullum bump :)

File: migrations/v10x/m3_add_schema.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ static public function depends_on()
 	/**
 	* Add table columns schema to the database:
 	*    boardrules:
-	*        rule_parents Basic rule data is serialzed and cached here
+	*        rule_parents Basic rule data is serialized and cached here
```
Comment (michaelcullum) [inline migrations/v10x/m3_add_schema.php]: serialised

**Comment by VSEphpbb** [comment]
Updated

**Pull request merged**